src/main/java/com/epam/lab/paymentsystem/service/impl/AccountServiceImpl.java

```diff
@@ -65,21 +59,34 @@ public Account createAccount(AccountDto accountDto) throws UnsupportedOperationE
    * @throws UnsupportedOperationException if there's not enough money on source account
    */
   @Override
-  @Transactional
   public void makeTransaction(Account source, Account target, long amount)
```

Comment:
https://stackoverflow.com/questions/30868829/is-it-possible-to-invoke-transactional-method-from-non-transactional-method-in-s